Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py

```diff
@@ -56,7 +56,7 @@ def _maybe_add_webkitcorepy_path():
 AutoInstall.register(Package('webkitbugspy', Version(0, 8, 0)), local=True)
 
 if sys.version_info > (3, 6):
-    AutoInstall.register(Package('rapidfuzz', Version(2, 11, 1), implicit_deps=['pyparsing']))
+    AutoInstall.register(Package('rapidfuzz', Version(2, 11, 1), implicit_deps=['pyparsing'], wheel=False))
```

Comment:
Everywhere else we use it, we have a comment saying _why_ we're using `wheel=False`. We should at least comment this, so we know _when_ we can remove the `wheel=False` argument. It seems like https://github.com/maxbachmann/RapidFuzz/commit/abcf645ec3a4563a0d90db56bd61389bec5ed60c should practically fix the universal macOS wheels, which seems like the problem.

Reply:
To copy/paste from elsewhere:

`rapidfuzz-2.11.1-cp38-cp38-macosx_10_9_universal2.whl` is broken, `rapidfuzz-2.11.1-cp38-cp38-macosx_10_9_x86_64.whl` seems fine.

Specifically, with the universal2 wheel, we get:

```
ImportError: dlopen(/Volumes/Data/worker/Style-EWS/build/Tools/Scripts/libraries/autoinstalled/python-3-x86_64/rapidfuzz/distance/metrics_cpp.cpython-38-darwin.so, 0x0002): symbol not found in flat namespace (__ZN4Avx215OsaDistanceInitEP14_RF_ScorerFuncPK10_RF_KwargsxPK10_RF_String)
```